### DISCUSSION TITLE
Volume options keeps on showing mid-gameplay in osu!catch

### DISCUSSION THREAD

**Discussion by Nekomosh004** [ppy/osu#19336] [Ideas] [answered]
Unlike Stable/CE, adjusting the volume in lazer keeps the options up when playing in osu!catch because we use the left and right arrow keys (by default) to control the catcher. I think its best described on these videos:

**What lazer do:**

https://user-images.githubusercontent.com/82471957/180611401-96b6ef9e-cc98-493f-8c34-fa8af575d8b6.mp4

**What Stable/Cutting Edge do:**

https://user-images.githubusercontent.com/82471957/180611626-3c92f15b-557d-450f-8665-dfef00881db6.mp4

I think its better if the options were auto-hidden mid any gameplay. Thank you for your attention on this! :D

**Comment by frenzibyte** [comment, marked as answer]
#19362
